backend/app/modules/uq/core.py

```diff
@@ -3,12 +3,15 @@
 from app.constants import UQ_THRESHOLD, LOW_UQ_SCORE, MAX_UQ_SCORE
 
 
-def calculate_uq(
-    gp_score: float, budget: int, user_address: str, addresses: list[str]
-) -> Decimal:
+def calculate_uq(gp_score: float) -> Decimal:
     if gp_score >= UQ_THRESHOLD:
         return MAX_UQ_SCORE
-    if budget > 0 and user_address not in addresses:
-        return MAX_UQ_SCORE
-    else:
-        return LOW_UQ_SCORE
+
+    return LOW_UQ_SCORE
+    # Previous logic by cheking the budget and addresses blacklist
```

Comment:
don't hesitate to remove the prior code snippets

Reply:
I can remove those, but AC assumed the following :)
```
The "has budget" logic (which automatically gives the user UQ score of 1 (multiplier)) should be commented out from the code so it stays there as a backup solution..
```